app.py

```diff
@@ -110,6 +110,27 @@
 logger.handlers[0].setFormatter(logging.Formatter(
 		fmt='%(asctime)s %(message)s', datefmt='%Y-%m-%d %H:%M:%S'))
 
+def refreshqueue(username):
+	""""Ensures that user can view annotations of sentences not in the 'initpriorities' queue.
+	These sentences are shown first, before the prioritized queue."""
+	db = getdb()
+	cur = db.execute(
+		'SELECT * FROM entries WHERE username = ? ORDER BY sentno ASC',
```

Comment:
If you are going to be accessing fields for each record by index (`row[1]`) probably best to specify the columns by name.